RCodes/08-11-21-Plots.R

```diff
@@ -0,0 +1,86 @@
+periods <- c("Aug17-Feb18"  ,"Mar18-July18" , "Aug18-Feb19", "Mar19-July19" , 
+             "Aug19-Feb20" , "Mar20-July20" , "Aug20-Feb21","Mar21-July21" 
+             , "Aug21-Feb22")
+
+months <-c("Jan","Feb","Mar",
+               "Apr","May","Jun",
+               "Jul","Aug","Sep",
+               "Oct","Nov","Dec")
+
+period.mon.data <- matrix(0,nrow = 59, ncol = 12)   
```

Comment:
why 59?
ncol=length(months) ?

Reply:
Updated the constants to the length of list.